src/materials/Material.js

```diff
@@ -62,7 +62,7 @@ class Material extends EventDispatcher {
 
 		this.dithering = false;
 
-		this.alphaTest = 0;
+		this.alphaTest = null;
```

Comment:
When I remember correctly, having a nullable material property seemed problematic when sheen was implemented. There is also a PR that tries to remove this behavior, see https://github.com/mrdoob/three.js/pull/17700/files#diff-98984bfa7fa23284a6a8bce191397059df3700f4377553fa7312dd69a78d780fL38.

The policy for sheen right now is that `null` disables its influence in the shader. With this change, this would also be `true` for `alphaTest`. I'm just mentioning this since #17700 should be closed if this PR is going to be merged.

Reply:
Instead of using `null`, we could keep the status quo and set `USE_ALPHATEST` only when `alphaTest` is greater zero. I guess I would prefer this approach.

Reply:
Yes, I've tried to make `null` work but having this double state is a nightmare, speciall in the editor...
Reverting to `0` but adding a `> 0` check.

Reply:
> Instead of using `null`, we could keep the status quo and set `USE_ALPHATEST` only when `alphaTest` is greater zero. I guess I would prefer this approach.  Introducing a uniform still makes sense since the engine produces less shader compilations when `alphaTest` is animated (although I'm not sure how common this use case is).

Ha! I missed that comment. Glad that we came to the same conclusion 😀

Reply:
> The policy for sheen right now is that `null` disables its influence in the shader. With this change, this would also be `true` for `alphaTest`. I'm just mentioning this since #17700 should be closed if this PR is going to be merged.

After working on this PR I think #17700 is the way to go indeed.

Reply:
I'll finish that PR after we're done with this one 👍